core/src/stories/patterns/navigation/navigation-basic.stories.tsx

```diff
@@ -76,7 +76,8 @@ const Template = () =>
       </tds-header-launcher-list>
     </tds-header-launcher>
   
-    <tds-header-brand-symbol slot="end" link-href="https://design.scania.com" aria-label="Scania - red gryphon on blue shield">
+    <tds-header-brand-symbol slot="end" aria-label="Scania - red gryphon on blue shield">
```

Comment:
The aria label should be moved to the `<a>` tag since that is the element with the image.